index.html

```diff
@@ -12,18 +12,19 @@
   <body>
 
     <div class="container roundbottom">
+      <!-- desktop menu -->
       <header class="navigation">
         <nav class="toolbar linkstyle">
           <div class="logo blue">
-            <a class="blue" href="#">My logo</a>
+            <a class="blue animation" href="#">My logo</a>
           </div>
           <ul class="menu liststyle linkstyle">
             <li><a class="onhover pressed gray" href="#">Portfolio</a></li>
             <li><a class="onhover pressed gray" href="#">About</a></li>
             <li><a class="onhover pressed gray" href="#">Contact</a></li>
           </ul>
         </nav>
```

Comment:
### [Optional] Kindly give some margin/padding on the top to make some room for the navbar. 
<img width="1440" alt="Screen Shot 2022-04-29 at 3 10 24 PM" src="https://user-images.githubusercontent.com/83048208/165925673-5698ef80-1569-40fb-96d6-c5a7529c0628.png">